src/main/drivers/adc_stm32h7xx.c

```diff
@@ -110,8 +110,14 @@ const adcTagMap_t adcTagMap[] = {
     // Keep these at the beginning for easy indexing by ADC_TAG_MAP_{VREFINT,TEMPSENSOR}
 #define ADC_TAG_MAP_VREFINT    0
 #define ADC_TAG_MAP_TEMPSENSOR 1
+
+#if defined(STM32H743xx) || defined(STM32H750xx) || defined(STM32H7A3xx) || defined(STM32H7A3xxQ)
+    { DEFIO_TAG_E__NONE, ADC_DEVICE_FOR_INTERNAL,   ADC_CHANNEL_VREFINT,    18 },
```

Comment:
This line seems to be common.

Reply:
Ah ... this is an artifact (?) of splitting independent arch support from a branch which contains multiple arches...
Next H7 support coming up is for H745/H747, and it has these lines
```
    { DEFIO_TAG_E__NONE, ADC_DEVICE_FOR_INTERNAL,   ADC_CHANNEL_VREFINT,    19 },
    { DEFIO_TAG_E__NONE, ADC_DEVICE_FOR_INTERNAL,   ADC_CHANNEL_TEMPSENSOR, 18 },
```
I would like to leave it as is, to minimize changes when these lines comes in.

Thoughts?

Reply:
Fine by me.